### PR TITLE
Update office-365-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line for clarity "Group Owners cannot access data shared in the group unless they are members."

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1819

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/create-groups/office-365-groups.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Any user can create a group unless you [limit group creation to a specific set o
 
 Groups have the following roles:
 
-- **Owners** - Group owners can add or remove members and have unique permissions like the ability to delete conversations from the shared inbox or change different settings about the group. Group owners can rename the group, update the description or picture and more.
+- **Owners** - Group owners can add or remove members and have unique permissions like the ability to delete conversations from the shared inbox or change different settings about the group. Group owners can rename the group, update the description or picture and more. Group Owners cannot access data shared in the group unless they are members.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/office-365-groups.md]:
```suggestion
- **Owners** - Group owners can add or remove members and have unique permissions like the ability to delete conversations from the shared inbox or change different settings about the group. Group owners can rename the group, update the description or picture, and more. Group Owners cannot access data shared in the group unless they are members.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #1819.

Thank you.

cc: @kenwith

**Comment by MikePlumleyMSFT** [comment]
@msbemba the statement is not true. I'm not aware of any unique permissions that members have that owners do not have.

**Comment by msbemba** [comment]
@MikePlumleyMSFT  The members receive emails or other items shared within the group. The owner receives it only if he is a member. There are situations when there is an admin who owns the group but doesn't really receive the messages or emails sent to the group.

Thanks
Sri

**Comment by mypil** [comment]
@MikePlumleyMSFT - We would like to follow up. Can you advise how you want to proceed with this PR?

Thank you.

**Comment by MikePlumleyMSFT** [comment]
Clarifying this requires further research. Will look at documenting this in a future update. Closing this for now.

**Pull request closed without merging**